packages/core/src/components/table/table-body-cell/table-body-cell.tsx

```diff
@@ -24,6 +24,9 @@ export class TdsTableBodyCell {
   /** Disables internal padding. Useful when passing other components to cell. */
   @Prop({ reflect: true }) disablePadding: boolean = false;
 
+  /** Setting for text align, no default value. Other accepted values are "left", "start", "right", "end" or "center". */
+  @Prop({ reflect: true }) textAlign?: 'left' | 'start' | 'right' | 'end' | 'center';
```

Comment:
I'm thinking, does it make sense to have a default value? As we are already defaulting to "left". As such we also avoid null in controls?

Reply:
The body cell will inherit the text alignment from the header cell anyway. So I just skipped it. But if the CoC prohibits nullable variables, then I can set `left` as default value. Sounds good?

Reply:
@AndreasRasovicScania, yes please set default/fallback value to left.